spec/requests/users_spec.rb

```diff
@@ -1,17 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe "Users", type: :request do
+  fixtures :users
   describe "GET /index" do
     it "returns http success" do
-      get "/users"
+      get users_path
       expect(assigns(:users)).to eq(User.all)
       expect(response).to have_http_status(:success)
     end
   end
 
   describe "GET /show" do
     it "returns http success" do
-      get "/users/show"
+      get user_path(users(:user1).id)
+      expect(assigns(:user)).to eq(users(:user1))
```

Comment:
Deixei passar isso aqui, perdão!

esse expect aqui testando se ta retornando o usuário correto poderia ser adicionado em um `it` separado, pq isso já não se encaixa muito no contexto desse teste de verificar se `"returns http success"`. 

Ele poderia ser adicionado em um, por exemplo, `it "returns the correct user"`.